### PR TITLE
docs: update `README.md` with project details

### LINKED ISSUE #353: Update `README.md` with project details
## Description

As our teacher requested, we need to update our `README.md` with the following details in the criteria.

## Criteria

- Include project description with objectives
- Provide usage examples
- Tech stack
- Add installation instructions for future developers
- Mention contributors

### PR DESCRIPTION
**Pull request by akiomatic**
## Overview

I've enhanced the `README.md` by adding comprehensive details about the project, including an overview and the tech stack. You can review the changes in the following link:

README: https://github.com/Tascurator/tascurator-frontend/blob/docs/353-update-readmemd-with-project-details/README.md

## Changes

- Updated `README.md` with our project details
- Moved the `Getting Started` section to `SETUP_GUIDE.md`

## Review points

Please take a look at the README and let me know if you have any suggestions or comments.

## Assignee Checklist:

- [x] The base branch is correct (no accidental merges)
- [x] The branch name follows our branch naming rules
- [x] The PR title follows our PR title rules
- [x] My code follows our coding style
- [x] I have commented on my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings or errors in the console

## Reviewer Checklist:

- [ ] Code readability and simplicity
- [ ] Follows best practices and coding standards
- [ ] Understandable and maintainable code

### CODE REVIEW

**Comment by KaiKoide** [comment]
@akiomatic 
Thank you for your work! It would be helpful if you could consider the following two points:

- Since there are no images, people who are unfamiliar with this application might not understand what is being explained. Having an image for each feature would make it easier to understand
- I think it would be beneficial to provide more explanation about the two roles, landlord and tenant

File: README.md [hunk 1]
```diff
@@ -1,63 +1,67 @@
...
-```
+## Key Features
 
-Second, run Docker container:
+### For landlords:
 
-```bash
-docker compose up
-```
+- **Task Management**: Create and assign tasks to tenants with ease.
```
Comment (yuki-92c) [inline README.md]: In my opinion, it's better to include "landlord can group multiple tasks together"

**Comment by akiomatic** [comment]
@KaiKoide Thank you for your review.

> Since there are no images, people who are unfamiliar with this application might not understand what is being explained. Having an image for each feature would make it easier to understand

I appreciate the suggestion. I apologize for not mentioning this in the PR description, but I actually plan to add images in a **separate** PR once the application is fully released.

> I think it would be beneficial to provide more explanation about the two roles, landlord and tenant

You make a good point. However, I intended for the mention of “…streamline the management of cleaning duties in a <ins>share house</ins>” to imply the presence of both landlord and tenant roles, as it’s common in that context. Additionally, since the terms “landlord” and “tenant” are used in this application with their usual meanings, I didn’t feel the need to provide further explanation. That said, if you think it would be clearer with the additional explanation, I’m happy to include it.

I've answered your comments. I'd like to hear your opinions! Thanks!

File: README.md [hunk 2]
```diff
@@ -1,63 +1,67 @@
-# Tascurator-frontend
+<div align="center">
+	<img src="public/logo.svg" alt="Tascurator Logo" width="120" />
+</div>
 
-## Getting Started
+<br>
 
-First, install the dependencies:
+**Tascurator** is a web application designed to streamline the management of cleaning duties in a share house.
```
Comment (yuki-92c) [inline README.md]: I think the term `share house` is not that commonly used, so either change the wording or use double quotation is better.

**Review by yuki-92c — COMMENTED** [review]
Thanks Akio! Please have a look at the comments I wrote.

**Comment by KaiKoide** [comment]
https://github.com/Tascurator/tascurator-frontend/pull/365#issuecomment-2287930410
@akiomatic 
Thank you for your reply.
I understand that the images will be addressed in a separate PR. After talking to an expert, I think it might be acceptable for the README to assume the people are familiar with tech. Therefore, I suggest including a few images that represent the key features of the app, rather than an image for each feature.

Regarding the roles, I’ve reconsidered and think that additional explanation may not be necessary, given that the Key Features section already distinguishes between landlord and tenant. Therefore, no further action is needed on that front.

**Review by Akikaze1119 — APPROVED** [review]
Thank you!
LGTM because my requests have already been reflected!

**Review by HrRn — APPROVED** [review]
Thank you for your great work :)

**Review by KaiKoide — APPROVED** [review]
LGTM!

**Review by yuki-92c — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**